src/utils/plotSpecs/generateEmbeddingContinuousSpec.js

```diff
@@ -27,7 +27,6 @@ const generateSpec = (config, plotData) => {
   }
   return {
     $schema: 'https://vega.github.io/schema/vega/v5.json',
-    description: 'A basic scatter plot example depicting gene expression in the context of UMAP.',
```

Comment:
I delete this because this is not appropriate anymore. This info is not shown in the UI - it is embedded into the `name` element of the plot. By default, the name is `Vega visualization`, and that is what is used to check in tests.